api/database/factories/PoolFactory.php

```diff
@@ -321,12 +321,23 @@ public function withBookmark(string $user_id)
         });
     }
 
-    // Add a team with at least one member to the pool
-    public function withTeamMembers()
+    /**
+     * Attach the users to the related pool.
+     * Creates a new user if no userIds passed in.
+     *
+     * @param array|null  $userIds - Id of the users to attach the role to
+     * @return void
+     */
+    public function withProcessOperators(array|null $userIds = null)
```

Comment:
I like how this is written. Maybe add similar methods to CommunityFactory?

Reply:
Thanks! Added community variants in [9c8d27d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10621/commits/9c8d27d4c5e0b85ca2c0a347c9cedb50ee61fc85).